src/scss/components/_content-header.scss

```diff
@@ -7,19 +7,39 @@
 
 .c-contentHeader {
     padding-top: spacing(x2);
-    padding-bottom: spacing(x2);
+    padding-bottom: spacing();
 }
 
 .c-contentHeader-title {
     padding: 0;
     display: inline-block;
     margin: 0 spacing(x2) 0 0;
+    font-weight: $font-weight-base;
+    font-family: $font-family-headings;
```

Comment:
Does this not also use a `h` tag?  Might be good to see example markup, as if it's a title probably should be a header semantically…?

Reply:
It does, was trying to make more generic but I agree with the header semantics so i've removed the font family :)